tests/test_release_supply_chain.py

```
@@ -0,0 +1,461 @@
+    assert "command -v syft" not in content
+    assert "${HOME}/.cache/apme/bin" in content
+    assert "SYFT_BIN" in content
+    assert "1.21.0 linux amd64" in checksum_content
+    assert "1.21.0 darwin arm64" in checksum_content
+    assert "SYFT_CHECKSUMS_FILE:-" not in content
+
+
+def test_install_syft_rejects_unsupported_release() -> None:
+    """Unsupported SYFT_VERSION must fail before any download."""
+    install_sh = REPO_ROOT / "containers" / "ci" / "install-syft.sh"
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)